salt/kube-apiserver/init.sls

```diff
@@ -47,13 +47,30 @@ kube-apiserver:
       - sls:             ca-cert
       - caasp_retriable: {{ pillar['ssl']['kube_apiserver_crt'] }}
       - x509:            {{ pillar['paths']['service_account_key'] }}
+      - file:            /etc/kubernetes/audit-policy.yaml
```

Comment:
You are forcing the need for this file, while it could be

```
{%- if salt.caasp_pillar.get('api:audit:enabled', False) %}
    - file:            /etc/kubernetes/audit-policy.yaml
{%- endif %}
```

Reply:
There's no harm in having this file around, even if the feature is disabled. I would prefer to leave the salt states as slim as possible.